pkg/cmd/pulumi/stack_change_secrets_provider.go

```diff
@@ -34,7 +34,7 @@ func newStackChangeSecretsProviderCmd() *cobra.Command {
 		Args:  cmdutil.ExactArgs(1),
 		Short: "Change the secrets provider for the current stack",
 		Long: "Change the secrets provider for the current stack. " +
-			"Valid secret providers types are `default`, `passphrase`, `awskms`, `azurekeyvault`, `gcpkms`, `hashivault`.\n\n" +
+			"Valid secret providers types are `default`, `passphrase`, `awskms`, `azurekeyvault`, `gcpkms`, `hashivault`, 'passthrough'.\n\n" +
```

Comment:
I would love to see this turned into a centrally stored list instead of copying these values around and hoping to remember them all for each update.